esp/services/ws_dfu/ws_dfuHelpers.cpp

```diff
@@ -183,6 +183,8 @@ bool WsDFUHelpers::addToLogicalFileList(IPropertyTree& file, const char* nodeGro
             if (!accessed.isEmpty())
                 lFile->setAccessed(accessed.replace('T', ' '));
         }
+        if (version >= 1.61)
+            lFile->setCost(file.getPropReal(getDFUQResultFieldName(DFUQRFcost)));
```

Comment:
question: what happens if the file is a super file? (or a super file with >1 subfile).
I don't think the stats will be available/returned, because that would require the server to have aggregated numDiskReads/numDiskWrites, and there'd be a combination of locations too..
NB: they may be available if it's a super of just 1 sub.

Worth double checking, but if needed attributes not available, it's not worth calling setCost here

Reply:
future: if above is true, it will be a known caveat that supers (poss. with >1 sub) do not show aggregate costs, but worth opening a JIRA for future enhancement.

Reply:
You're absolutely right. Super files aren't supported at the moment.  I'll open a Jira to look into this later.

Reply:
for reference, can you reference the new JIRA here and/or link it to this PR's JIRA?

Reply:
Superfile costs: https://track.hpccsystems.com/browse/HPCC-26871